### PR TITLE
feat: create launcher for core services

### LINKED ISSUE #1578: Local launcher method for core services
Now that we also have core services, we should enable them through a dedicated launcher function for whenever they are installed locally (and not just through Docker containers)

### PR DESCRIPTION
**Pull request by RobPasMue**
## Description
Provide a launching mechanism for Core Services. Should work both on Windows and Linux

## Issue linked
Closes #1578 

## Checklist
- [x] I have tested my changes locally.
- [ ] I have added necessary documentation or updated existing documentation.
- [x] I have followed the coding style guidelines of this project.
- [ ] I have added appropriate unit tests.
- [x] I have reviewed my changes before submitting this pull request.
- [x] I have linked the issue or issues that are solved to the PR if any.
- [x] I have assigned this PR to myself.
- [x] I have added the minimum version decorator to any new backend method implemented.
- [x] I have made sure that the title of my PR follows [Conventional commits style](https://www.conventionalcommits.org/en/v1.0.0/#summary) (e.g. ``feat: extrude circle to cylinder``)

### CODE REVIEW

**Comment by RobPasMue** [comment]
**IMPORTANT NOTE** - when using ``ansys-tools-path`` on Linux, it is important that the Geometry Core Service is located in one of the following directories:

* ``/usr/ansys_inc/vXXX/CoreGeometryService``
* ``/install/ansys_inc/vXXX/CoreGeometryService``
* ``/ansys_inc/vXXX/CoreGeometryService``

Feel free to use symbolic links to try this out.

File: src/ansys/geometry/core/connection/product_instance.py [hunk 1]
```diff
@@ -348,6 +361,77 @@ def prepare_and_start_backend(
...
+                )
+            )
+        else:
+            # Verify dotnet is installed
+            import shutil
+
+            if not shutil.which("dotnet"):
+                raise RuntimeError(
+                    "Cannot find 'dotnet' command. "
+                    "Please install 'dotnet' to use the Ansys Geometry Core Service."
```
Comment (MaxJPRey) [inline src/ansys/geometry/core/connection/product_instance.py:407]: Maybe here we could mention that at least 8.0 version is required. So people don't install an older version the first time and have to do it again.

**Review by MaxJPRey — DISMISSED** [review]

File: src/ansys/geometry/core/connection/product_instance.py (hunk 1 above)
Comment (RobPasMue) [reply src/ansys/geometry/core/connection/product_instance.py:407]: That's actually a good idea. I'll do that, thanks @MaxJPRey

**Comment by RobPasMue** [comment]
Let's merge this one as is. Testing for launching local products is not in yet anyway (our VMs rely on Docker). We can come back to fix things.

**Review by b-matteo — APPROVED** [review]

**Pull request merged**